hercules-ci-agent/CHANGELOG.md

```diff
@@ -7,6 +7,34 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### [0.6.1] - 2019-11-06
+
+### Fixed
+
+ - Fix token leak to log caused by a library upgrade.
```

Comment:
If I understand correctly the leak happened only when printing exceptions, maybe it's worth saying that it's not always the case.

Reply:
```suggestion
 - Fix token leak to system log when reporting an HTTP exception. This was introduced by a library upgrade.
```